drivers/hmis/app/views/hmis_client/assessments/show.haml

```diff
@@ -36,5 +36,11 @@
       %dl
         %dt= label
         %dd= value
-- else
+- elsif !@assessment.hud_assessment?
   .none-found No questions provided.
+-# Always link to HMIS for HUD Assessments, since the majority of data is non-Custom so wont be shown here.
+- if @assessment.hud_assessment?
+  .none-found
+    %a{href: @assessment.data_source.hmis_url_for(@assessment), target: :blank}
+      Open in HMIS
+      %i.icon-link-ext.ml-2{style: 'color: inherit; font-size: inherit;'}
```

Comment:
There typically won't be any data shown for HUD Assessments, so I included this link. I could take or leave it, though. If the Intake Assessment has any custom fields, they would be shown above as well.

Another option is to just not link to a modal for HUD Assessments.  I'm fine either way .

<img width="1223" alt="Screenshot 2024-03-22 at 10 01 31 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/0c9440a7-7662-4689-bbcd-58deaa0bcd86">

Reply:
I think that works well, thanks!